src/js/utils/CleanJSONUtil.js

```diff
@@ -0,0 +1,20 @@
+import JobConfig from '../constants/JobConfig';
+import ServiceConfig from '../constants/ServiceConfig';
+
+export function cleanJSONspec(jsonSpec) {
```

Comment:
Can we rename these utils so that it's obvious that this one is about jobs and the other one about services?